skill-tree /.github/workflows/ci.yml

```diff
@@ -0,0 +1,22 @@
+name: CI Build
+
+on:
+  pull_request:
+    branches:
+      - '**'
+jobs:
+  build:
+    name: Maven Build
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+
+      - name: Setup Java 17
+        uses: actions/setup-java@v3
+        with:
+          java-version: 17
+          distribution: 'temurin'
+          cache: 'maven'
+
+      - name: Build with Maven
+        run: ../.././mvnw verify --file ../../pom.xml
```

Comment:
- does this also run tests as suggested in the title of this pr?

Reply:
Yes it does

Reply:
@yesyash I wanted to understand if the relative folder path given here is a proper format? Have not written the action script before.

Reply:
- it looks good to me on a first glance, if you want to test the action locally you can use a tool called (act)[https://github.com/nektos/act]